src/Uno.UI/Uno.UI.csproj

```diff
@@ -172,6 +172,7 @@
 	<ItemGroup>
 		<Compile Include="Mixins\Android\BaseActivity.Callbacks.g.cs" />
 		<Compile Include="Mixins\Android\FrameworkElementMixins.g.cs" />
+		<Compile Include="Mixins\Android\FrameworkElementMixins.g.cs" />
```

Comment:
Duplicate lines
- 3 copies of "Mixins\Android\FrameworkElementMixins.g.cs"
- 2 copies of "Mixins\DependencyPropertyMixins.g.cs"